sdk/core/Azure.Core.TestFramework/src/Instrumentation/DiagnosticScopeValidatingInterceptor.cs

```diff
@@ -209,9 +209,13 @@ internal static async ValueTask<T> ValidateDiagnosticScope<T>(Func<ValueTask<(T
                     }
                     else
                     {
-                        if (!diagnosticListener.Scopes.Any())
+                        // If ForwardsClientCallsAttribute is being used on the method, we don't know what the name of the scope should be because there could be many
+                        // differently named methods sharing the same scope name, but we do know that there should be some scope created other than the Azure.Core scope.
+                        if (!diagnosticListener.Scopes.Any(e => !e.Name.StartsWith("Azure.Core")))
                         {
-                            throw new InvalidOperationException($"Expected some diagnostic scopes to be created, found none");
+                            throw new InvalidOperationException(
+                                $"Expected some diagnostic scopes to be created other than the Azure.Core scopes, but no such scopes were present. {Environment.NewLine} Ensure that the inner method that " +
```

Comment:
Nit: remove the space after the new line (alignment will be wonky otherwise), or just remove the new line entirely. It's rare to see or need formatted exception message.